storage/FileManagerPosix.cpp

```diff
@@ -86,7 +88,7 @@ size_t FileManagerPosix::numSlots(const block_id block) const {
   }
 
   if ((file_stat.st_size % kSlotSizeBytes) != 0) {
-    throw CorruptPersistentStorage();
+    LOG(FATAL) << "The file " << filename << " was corrupted.";
```

Comment:
Actually, I am thinking the following implementations in all `FileManager` versions:

```
DCHECK_EQ(0, file_stat.st_size % kSlotSizeBytes)
  << "The file " << filename << " was corrupted.";
```